app/controllers/users_controller.rb

```diff
@@ -34,6 +36,18 @@ def judge
 
   private
 
+  def css_id
+    @css_id ||= valid_css_id_or_nil
+  end
+
+  def valid_css_id_or_nil
+    return nil unless params[:css_id].presence
+
+    return to_valid_css_id(params[:css_id]) if invalid_css_id?(params[:css_id])
```

Comment:
Uppercasing of CSS_ID is already done by the User model https://github.com/department-of-veterans-affairs/caseflow/blob/57e746e91b91c82c9b5f4edac343bde9688b68d9/app/models/user.rb#L491
but we'll do it in the controller to correct the CSS_ID ASAP since `css_id` is used in many places in the controller.

Reply:
Think the `invalid_css_id?` check here is probably just as costly as blindly upcasing all css_ids. Can just return all upcased css_ids here.
```suggestion
    to_valid_css_id(params[:css_id])
```